code/datums/migrants/migrant_wave.dm

```diff
@@ -37,135 +37,33 @@
-	downgrade_wave = /datum/migrant_wave/adventurer_down_two
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/adventurer = 3,
-	)
-	greet_text = "Together with a party of trusted friends we decided to venture out, seeking thrills, glory and treasure, ending up in the misty and damp bog underneath Vanderlin, perhaps getting ourselves into more than what we bargained for."
-
-/datum/migrant_wave/adventurer_down_two
-	name = "Adventure Party"
-	downgrade_wave = /datum/migrant_wave/adventurer_down_three
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/adventurer = 2,
-	)
-	greet_text = "Together with a party of trusted friends we decided to venture out, seeking thrills, glory and treasure, ending up in the misty and damp bog underneath Vanderlin, perhaps getting ourselves into more than what we bargained for."
-
-/datum/migrant_wave/adventurer_down_three
-	name = "Adventure Party"
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/adventurer = 1,
-	)
-	greet_text = "Together with a party of trusted friends we decided to venture out, seeking thrills, glory and treasure, ending up in the misty and damp bog underneath Vanderlin, perhaps getting ourselves into more than what we bargained for."
 
 /datum/migrant_wave/bandit
 	name = "Bandit Raid"
 	spawn_landmark = "Bandit"
-	downgrade_wave = /datum/migrant_wave/bandit_down_one
 	weight = 8
 	roles = list(
 		/datum/migrant_role/bandit = 4,
 	)
 	can_roll = FALSE
 
-/datum/migrant_wave/bandit_down_one
-	name = "Bandit Raid"
-	spawn_landmark = "Bandit"
-	downgrade_wave = /datum/migrant_wave/bandit_down_two
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/bandit = 3,
-	)
-
-/datum/migrant_wave/bandit_down_two
-	name = "Bandit Raid"
-	spawn_landmark = "Bandit"
-	downgrade_wave = /datum/migrant_wave/bandit_down_three
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/bandit = 2,
-	)
-
-/datum/migrant_wave/bandit_down_three
-	name = "Bandit Raid"
-	spawn_landmark = "Bandit"
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/bandit = 1,
-	)
-
 /datum/migrant_wave/merc
 	name = "Band of Mercenaries"
-	downgrade_wave = /datum/migrant_wave/merc_down_one
 	weight = 8
 	roles = list(
 		/datum/migrant_role/mercenary = 4,
 	)
-
-/datum/migrant_wave/merc_down_one
-	name = "Band of Mercenaries"
-	downgrade_wave = /datum/migrant_wave/merc_down_two
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/mercenary = 3,
-	)
-
-/datum/migrant_wave/merc_down_two
-	name = "Band of Mercenaries"
-	downgrade_wave = /datum/migrant_wave/merc_down_three
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/mercenary = 2,
-	)
-
-/datum/migrant_wave/merc_down_three
-	name = "Band of Mercenaries"
-	can_roll = FALSE
-	roles = list(
-		/datum/migrant_role/mercenary = 1,
-	)
+	greet_text = "A group of mercenaries, hired by the Vanderlin nobility to protect the city from threats, has arrived. They are well equipped and ready to face any challenge that comes their way."
```

Comment:
mercs are not guards . they are mercenaries i would recomend a different greet

Reply:
perhaps something like " a group of mercenary arriving to vanderlin looking for coin or smth"